### PR TITLE
Feat/andrew/sqlite backend merge into dev

### LINKED ISSUE #11: [US #5] All Tasks View
## Description

Have a webpage that takes all the tasks from Localstorage and renders them with sorting functionalities mentioned in #5 

## Frontend

Generate a webpage that is initially ordered by **Due Date** and has dropdown for sorting based on **Priority and Label**
## Backend

Write a route called `api/task/getAllTasks` that returns all Task Objects from Local Storage

### PR DESCRIPTION
**Pull request by andronius-yang**
## Tracking Info

Resolves #11

## Changes

Integrated all-tasks view with filter as a secondary page. Displays a list of tasks in chronological order. Integrated delete, edit functionality in weekly view.

## Testing

Normal standard database test to ensure sanity of database backend.

Hand testing UI: followed the procedures of creating multiple tasks to overfill all-tasks page, creating less tasks to see how the UI fits tasks. 

## Confirmation of Change

image

image

### CODE REVIEW

File: source/project/components/all-tasks/all-tasks.html
Comment (SamvritSrinath) [inline source/project/components/all-tasks/all-tasks.html:1]: Ideally, we don't need the "Due" Preview in the All Tasks view, but we can get this cleaned up by Saturday.

File: source/project/components/mainview/mainview.html
Comment (SamvritSrinath) [inline source/project/components/mainview/mainview.html:1]: Just a Temp Fix for The buttons

**Review by SamvritSrinath — APPROVED** [review]
Looks good enough to release! Just some bandaid fixes for Menu Button and we need to revise the preview/formatting of the boxes but good enough!

**Review by mialyssa — APPROVED** [review]
Works as intended! The temporary changes work well for now, can update once we add vault view. Good job!

**Pull request merged**